modules/luci-base/htdocs/luci-static/resources/tools/widgets.js

```diff
@@ -580,6 +580,7 @@ var CBIUserSelect = form.ListValue.extend({
 
 	load: function(section_id) {
 		return getUsers().then(L.bind(function(users) {
+			this.keylist = 0;
```

Comment:
Don't forget about `vallist` - it needs to be cleared as well. Also use `delete`, it is more explicit and feels less hackish:

```
delete this.keylist;
delete this.vallist;
```

Reply:
@jow- Updated according your suggestion